src/main/java/httpServer/RequestParserHelper.java

```diff
@@ -0,0 +1,18 @@
+package httpServer;
+
+import static httpServer.http.Constants.CRLF;
+
+public class RequestParserHelper {
+
+    public boolean isRequestLine(String line) {
+        if (line.contains(": ")) {
+            return false;
+        }
+        return true;
+    }
+
+    public boolean atEndOfHeaders(StringBuilder requestScannedInSoFar) {
```

Comment:
Again, while I do like the expressive variable naming, I think it's gotten to the point where it's actually less clarifying. Is there a clearer name than `requestScannedInSoFar`?

Reply:
Not sure - is `httpRequest` on its own better? Or something like `httpRequestBuilder`?